catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -103,12 +117,14 @@ public String toString() {
   @Override
   public List<WorkspaceMetacardImpl> getWorkspaceMetacards() {
 
-    final QueryRequest queryRequest = createQueryRequestForAllWorkspaceMetacards();
+    final QueryRequest queryRequest = createQueryRequestForSubscribedToWorkspaces();
 
-    try {
-      return createWorkspaceMetacards(catalogFramework.query(queryRequest));
-    } catch (UnsupportedQueryException | FederationException | SourceUnavailableException e) {
-      LOGGER.warn("Error querying for workspaces", e);
+    if (queryRequest != null) {
+      try {
+        return createWorkspaceMetacards(catalogFramework.query(queryRequest));
+      } catch (UnsupportedQueryException | FederationException | SourceUnavailableException e) {
+        LOGGER.warn("Error querying for workspaces", e);
```

Comment:
❓  Do we want these to be at `warn` ? seems like these might pollute the log if a source is down or something.